_posts/2022-11-17-what-makes-a-python-package-healthy.md

```diff
@@ -0,0 +1,536 @@
+
+Alas it is true that metrics designed for reporting that a funder requires
+for a grant may differ metrics designed fom internal evaluation that 
+informs program development. pyOpenSci has a lot to unpack there over the 
+upcoming months!
+
+### Three open source software healthy metric "buckets" 
+
+Based on all of the Twitter feedback (below), and what I *think* might be a 
+start at what pyOpenSci needs, I organized the Twitter conversation into three 
+buckets:
+
+1. Infrastructure
+2. Maintenance 
+3. Community adoption (and usability??)
+
+These three buckets are all priorities of pyOpenSci.
+
+<i class="far fa-sticky-note"></i> DEIA is another critical concern for pyOpenSci but I won't discuss that in this blog post. 
+{: .notice }
+
+
+## Infrastructure in a Python open source GitHub repository as a measure of package health
+
+So here I start with Python package infrastructure found in a GitHub repository 
+as a preliminary measure of package health. 
+When think of infrastructure I think about the files and "things" available in a 
+repository that support it's use. I know that no bucket is perfectly isolated
+from the others but i'm taking a stab at this here. 
+
+<i class="fas fa-info-circle"></i> The code for many open source software packages can be found on [GitHub](https://www.github.com). GitHub is a free-to-use website that runs `git` which is a version control system. Version control allows developers to track historical changes to code and files. GitHub which is a website allows developers to communicate openly, review new code changes and update content in a structured way. 
+{: .notice }
+
+### What does GitHub (and Ivan) think about health checks for Python open source software?
+
+Ivan, is a long time advisor, editor and member of the pyOpenSci community. He's also a generally a great human being who is growing open science efforts such as 
+[Open Science Labs](https://opensciencelabs.org/); which is a gloabl community devoted to education efforts and tools that support open science.  
+
+Ivan was quick to point out some basic metrics offered by GitHub which follow their 
+[community standards online guidebook here](https://opensource.guide/).
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Maybe not totally related, but github has a section called community standards that mabe could be used as reference, for example: <a href="https://t.co/wmu1bDdcQR">https://t.co/wmu1bDdcQR</a></p>&mdash; XMN (@xmnlab) <a href="https://twitter.com/xmnlab/status/1578745372880429057?ref_src=twsrc%5Etfw">October 8, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
+
+Actually it's totally related, Ivan! Let's have a look look at the [pyOpenSci contributing-guide GitHub repository](https://www.github.com/pyopensci/contributing-guide/community) to see how we are 
```

Comment:
This link should be peer-review-guide now right?
I couldn't find the page you meant to link to, though

Reply:
it's the right link if you just use peer-review instead of contributing. it is the community checks page in github that all repos have.

Reply:
are you not able to see it? i didn't think it needed permissions to view if it's an open repo